core/app/models/spree/order_updater.rb

```diff
@@ -21,6 +21,7 @@ def update
         update_payment_state
         update_shipments
         update_shipment_state
+        update_promotions_usage_count
```

Comment:
I don't believe this will be a good time to do this. Order update can be called somewhat often and this is probably something we want to do rarely.

Order updater also isn't called immediately on `order.finalize!`, which may cause this to be incorrect.